.github/workflows/build.yml

```
@@ -0,0 +1,35 @@
+name: build
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*-[0-9]+.*"
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'zulu'
+          cache: 'maven'
+      - name: Set up CI environment
+        run: .github/setup.sh
+      - name: Execute the build
+        run: .github/build.sh
```

So let's add blosc and xvfb back:

```suggestion
      - name: Set up CI environment
        run: .github/setup.sh
      - name: Install blosc
        run: sudo apt install libblosc-dev
      - name: Install xvfb
        run: sudo apt-get install xvfb
      - name: Execute the build
        run: .github/build.sh
```